dex/testing/eth/create-node.sh

```diff
@@ -32,7 +32,7 @@ chmod +x "${NODES_ROOT}/harness-ctl/${NAME}"
 
 HTTP_OPT=""
 if [ "$NAME" = "delta" ]; then
-  HTTP_OPT="--http --http.port ${DELTA_HTTP_PORT} --ws --ws.port ${DELTA_WS_PORT} --ws.api \"db,eth,net,web3,personal,txpool,admin\""
+  HTTP_OPT="--http --http.port ${DELTA_HTTP_PORT} --ws --ws.port ${DELTA_WS_PORT} --ws.api \"eth,net,web3,personal,txpool,admin\""
```

Comment:
`db` was not a thing: `ERROR[01-26|14:00:13.226] Unavailable modules in HTTP API list     unavailable=[db] available="[admin debug web3 eth txpool personal clique net les vflux engine]"`